codersdk/deployment.go

```diff
@@ -1265,6 +1306,29 @@ when required by your organization's security policy.`,
 			Group:         &deploymentGroupConfig,
 			Value:         &c.Config,
 		},
+		{
+			Name:        "CLI SSH Hostname Prefix",
+			Description: "The CLI SSH deployment name is the used in the Hostname of the ssh config.",
```

Comment:
```suggestion
			Description: "The SSH deployment prefix is used in the Host of the ssh config.",
```
(Also Hostname => Host because that's what defines the name you type when connecting.)

Reply:
Ah ok.